### DISCUSSION TITLE
matrix room

### DISCUSSION THREAD

**Discussion by stsp** [dosemu2/dosemu2#1942] [General]
Hi, created a matrix room:
https://app.element.io/#/room/#dosemu2:matrix.org

Invited @bartoldeman and @stuaxo - other
guys seems to be not there, and @skitt there
is probably not our @skitt (not sure).

Its just out of nothing to do, we can chat also
here on github. But if anyone wants - please
join, and maybe eventually we'll find that more
useful than to talk here on github. Or maybe not.
Thoughts? :)

**Comment by stuaxo** [comment]
Let's see how it goes, I've got a different user there so just hopped on.   Sometimes I miss things on github when I'm busy with other bits, so matrix works for me.

**Comment by stsp** [comment]
Lets see.
We probably can mirror some mail-lists
there, although all dosemu-related MLs
are long ago dead. :)
Or maybe some people like @mcericdos
that really hate github, can join there.
Overall I think this all depends on how
many people willing to talk we have.
Currently I am not aware of too much. :)

**Reply by stuaxo** [reply]
It's good gitter moving over to a matrix client - gitter was just one extra thing I never got round to checking, matrix is general enough I have it open for other things already.
